src/parser/lexer.rs

```diff
@@ -160,8 +169,8 @@ pub enum NormalToken<'input> {
     Underscore,
     #[regex("m(%+)\"", |lex| lex.slice().len())]
     MultiStringStart(usize),
-    #[regex("s(%+)\"", |lex| lex.slice().len())]
-    SymbolicStringStart(usize),
+    #[regex("[a-zA-Z][_a-zA-Z0-9-']*-s(%+)\"", symbolic_string_prefix_and_length)]
+    SymbolicStringStart((&'input str, usize)),
```

Comment:
This is a bit weird, but logos does not support enum variants with more than one argument. And I didn't feel like making a new record for this single usecase.

Reply:
It's a detail, but suing a struct for a tuple doesn't sound too bad. If only for the explicit naming of arguments, which is self-documenting.